solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -81,6 +81,7 @@
 public class SolrCloudTestCase extends SolrTestCaseJ4 {
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  public static final Boolean USE_PER_REPLICA_STATE = Boolean.parseBoolean(System.getProperty("use.per-replica", "false"));
```

Comment:
Can we add a setting to the client to default whether it uses new state or not, and then that should simplify a lot of unit tests? our mini cluster can set it on the client for you?

Reply:
Yeah we need to find a simpler way to do this. 

But the client cannot be modified

Reply:
Can it be a setting in cluster properties that controls how new collections are created?

Reply:
There's a `defaults` section in our unofficial kitchen-sink ZK config file, i.e. `/clusterprops.json`.